common.cake

```diff
@@ -7,7 +7,13 @@
 #addin nuget:?package=Cake.Yaml&version=2.1.0
 #addin nuget:?package=Newtonsoft.Json&version=9.0.1
 #addin nuget:?package=Cake.Json&version=3.0.1
-#addin nuget:?package=Mono.ApiTools.NuGetDiff&version=1.0.1&loaddependencies=true
+#addin nuget:?package=Mono.ApiTools.NuGetDiff&version=1.0.2-preview.4&loaddependencies=true
+
+// if using the out-of-process nuget client:
+#addin nuget:?package=NuGet.Packaging&version=4.6.0
+#addin nuget:?package=NuGet.Versioning&version=4.6.0
+#addin nuget:?package=NuGet.Frameworks&version=4.6.0
+#addin nuget:?package=NuGet.Packaging.Core&version=4.6.0
```

Comment:
As a result of the interim solution of using the nuget.exe, we need this as well.